bootstrap.py

```diff
@@ -459,9 +459,15 @@ def delete_json(url):
     return call_api(url, method='DELETE')
 
 
-def put_json(url):
+def put_json(url, jdata=None):
     """Use `call_api` to place a "PUT" REST API call."""
-    return call_api(url, method='PUT')
+    return call_api(url, data=jdata, method='PUT')
+
+
+def update_host_capsule_mapping(server, port, attribute, capsule_id, host_id):
```

Comment:
We usually don't have `server` and `port` in the other functions, and just default to the global stuff. While not the nicest way, it would be consistent. I am torn.